src/Logs/LogsAggregator.php

```diff
@@ -149,7 +149,7 @@ public function add(
 
         // We check if it's a `LogsLogger` to avoid a infinite loop where the logger is logging the logs it's writing
         if ($sdkLogger !== null) {
-            $sdkLogger->log((string) $log->getLevel(), "Logs item: {$log->getBody()}", $log->attributes()->toSimpleArray());
+            $sdkLogger->log($log->getPsrLevel(), "Logs item: {$log->getBody()}", $log->attributes()->toSimpleArray());
```

Comment:
What (I think) @mfb points out is that we copy the log level from the log item instead of logging always as "DEBUG" or "INFO" and mentioning the log level of the log item in the log message.

This was introduced by me I think, this maybe should be updated to mention the log item level in the log message and set our log level to debug (?).

In addition, the comment above this `if` is outdated and incorrect :) The `LogsLogger` never made it in but the comment remains 🙈

Reply:
In that case I did this very by the book 😎 

Then maybe the comment could be updated but no other changes are needed.

Reply:
Ok, great - as long as it's by design :)

It's not 100% clear to me but I guess "appropriate" is saying the Sentry log level should be translated to an appropriate level, e.g. log a "fatal" to the console as "critical".  I'd also find it reasonable to just print the "fatal" level as part of an "info" log message - either way works for me.